test/base/src/androidTest/kotlin/io/realm/test/shared/QueryTests.kt

```diff
@@ -676,9 +807,131 @@ class QueryTests {
     }
 
     @Test
-    @Ignore
+    fun sum_find_emptyTable() {
+        val assertions = { propertyDescriptor: PropertyDescriptor ->
+            realm.query(QuerySample::class)
+                .sum(propertyDescriptor.property.name, propertyDescriptor.clazz)
+                .find { sumValue: Any? ->
+                    if (sumValue is Number) {
+                        assertEquals(0, sumValue.toInt())
```

Comment:
Shouldn't `sum` also return `null` if there are no entries? I would think the only aggregate that always have a value is `count` since that is the only one that doesn't depend on the values of the entries?